packages/cli-e2e/__tests__/vue.specs.ts

```diff
@@ -29,8 +38,52 @@ describe('ui', () => {
 
     beforeAll(async () => {
       browser = await getNewBrowser();
-      await setupUIProject('ui:create:vue', projectName, cliProcesses);
-    }, 240e3);
+      const projectPath = getProjectPath(projectName);
+      mkdirSync(projectPath, {recursive: true});
+      writeFileSync(
+        join(projectPath, '.yarnrc'),
+        'registry "http://verdaccio:4873"'
+      );
+      const buildProcess = setupUIProject('ui:create:vue', projectName);
+      cliProcesses.push(buildProcess);
+
+      buildProcess.stdout.on('data', async (data) => {
+        if (
+          /Pick an action: \(Use arrow keys\)/.test(stripAnsi(data.toString()))
+        ) {
+          await answerPrompt('\u001b[B', buildProcess);
+          return;
+        }
+        if (/❯ Merge/.test(stripAnsi(data.toString()))) {
```

Comment:
I'm not sure about this for a couple of reasons:
* I think we should always overwrite the project. Merging could lead to unpredicted results
* Instead of expecting a bunch of prompts, we could simply test the default flow with the flags `--default` and `--force` using the passthrough strategy

Also, I did not include these prompts in the tests because I assumed the container running the E2E tests would always be initialized to a clean state.

Reply:
What's going on with the Vue tests here:
 1. We create the directory where the project
 2. We write a `.yarnrc` that tells Yarn to uses Verdaccio
 
 Given those two, if we do not merge we will discard the `.yarnrc` and thus vue-cli won't use our registry.
 
 Therefore, without changing the behavior of any command, this is the only solution I found which is 'sound'
 (I could probably also spoof npm at the DNS level but... it's a bit more involved imo)

Reply:
- [ ] Double check if we can do without the local `.yarnrc`